packages/landing/src/map.ts

```diff
@@ -184,7 +186,7 @@ export class Basemaps {
             .transform(center, `EPSG:${this.config.projection}`, UrlProjection)
             .map((c: number) => Math.floor(c * 10e7) / 10e7); // Limit to 5 decimal places (TODO is 5 enough)
 
-        const zoom = Math.floor(view.getZoom() * 10e3) / 10e3;
+        const zoom = Math.floor(view.getZoom() ?? 0 * 10e3) / 10e3;
```

Comment:
how does `??` relate to `*`

Reply:
badly by the looks of the transpiled code! Well spotted!!